### PR TITLE
Added Differential Calc

### LINKED ISSUE #76: Differential Calculator
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

This calculator evaluates the Differential equations like sin(x) into cos(x)
Can your assign me this issue under SWOC'24 

### Screenshots

Screenshot 2024-01-03 214322

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

### PR DESCRIPTION
**Pull request by AbhineshJha**
Closes #76 

# Description👨‍💻 

It is a type of calculator which evaluates the derivatives 
# Type of change📄

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)

# How this has been tested✅

# Checklist✅ 

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have added demonstration in the form of GIF/video file
- [] I am an Open Source Contributor

# Screenshots/GIF📷
Screenshot 2024-01-03 223740

### CODE REVIEW

**Comment by Rakesh9100** [comment]
Pls add folder name as `Differential-Calculator`
Add a README.md file in your folder.
Also add your calculator section in main `index.html` file @AbhineshJha

**Comment by AbhineshJha** [comment]
done sir @Rakesh9100

File: Calculators/Differencial Calculator/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Technologies Used
+
+- HTML
+- CSS
+- JavaScript
+- [math.js](https://mathjs.org/) for symbolic computation
+
+## Getting Started
+
```
Comment (Rakesh9100) [inline Calculators/Differencial Calculator/README.md]: Pls delete this getting started section and add Screenshot of the calculator in this README @AbhineshJha

**Comment by AbhineshJha** [comment]
please review sir @Rakesh9100

**Comment by Rakesh9100** [comment]
image

### It is appearing like this. I suppose you have added the box section in `index.html` after the last div but you have to add inside this div `<div class="container" id="calculators">`. Pls check @AbhineshJha

**Comment by AbhineshJha** [comment]
fixed

**Comment by Rakesh9100** [comment]
Good job, Merged!! Congrats on getting your PR merged to this repo under SWOC S4 🎉🎉 @AbhineshJha

**Comment by AbhineshJha** [comment]
thanks a lot sir ! its my first pr in SWOC'24

**Pull request merged**